lib/kitchen/directions/bake_notes/bake_numbered_note.rb

```diff
@@ -3,7 +3,15 @@
 module Kitchen
   module Directions
     module BakeNumberedNote
-      def self.v1(note:)
+      def self.v1(book:, selectors:)
```

Comment:
ditto here on calling it `classes` because `$.#{selector}` assumes it is a class